gumberoo_backend/config/settings/base.py

```diff
@@ -56,6 +59,13 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:3000",
+    "http://127.0.0.1:3000",
+    "http://localhost:8000",
+    "http://127.0.0.1:8000"
+]
+
```

Comment:
Still not quite sure what CORS is but is there a reason the FE actual development website not listed

Reply:
@melatran Because in theory a development site should be able to access our site just fine. The CORS thing is related to cross resource sharing, so for security reasons localhost servers are blocked. At least how I understand it